Userland/Libraries/LibJS/AST.cpp

```diff
@@ -3537,7 +3537,7 @@ Completion MetaProperty::execute(Interpreter& interpreter) const
             interpreter.vm().host_finalize_import_meta(import_meta, module);
 
             // e. Set module.[[ImportMeta]] to importMeta.
-            module.set_import_meta({}, import_meta);
+            module.set_import_meta(import_meta);
```

Comment:
...and then call it like so:
```suggestion
            module.set_import_meta(Badge<MetaProperty> {}, import_meta);
```